collections/Laud_Misc/MS_Laud_Misc_280.xml

```diff
@@ -50,93 +45,174 @@
+                  <acquisition when="1635">Part of his first donation to the Bodleian, 1635.</acquisition>
                </history>
                <additional>
                   <adminInfo>
                      <recordHist>
                         <source><ref target="https://medieval.bodleian.ox.ac.uk/about">Summary description</ref> abbreviated from the Quarto Catalogue (H. O. Coxe,  <title>Laudian Manuscripts</title>, Quarto Catalogues II, repr. with corrections, 1969, from the original ed. of 1858-1885). 
-                            
-                            
-                            <listBibl>
-                              <bibl type="QUARTO" facs="aby0163.gif">Quarto Catalogue, col. 230</bibl>
-                              <bibl type="SC" facs="aap0073.gif">Summary Catalogue, vol. 2, part 1, p. 52</bibl>
-                           </listBibl>
+                           
+                           
+                           <listBibl><bibl type="QUARTO" facs="aby0163.gif">Quarto Catalogue, col. 230</bibl><bibl type="SC" facs="aap0073.gif">Summary Catalogue, vol. 2, part 1, p. 52</bibl></listBibl>
                         </source>
                      </recordHist>
                   </adminInfo>
                </additional>
             </msDesc>
          </sourceDesc>
-      </fileDesc>       
-      <revisionDesc>          <change when="2017-07-01">First online publication.</change>
-
-         <change when="2017-05-29">
-            <persName>James Cummings</persName> Up-converted the markup using <ref target="https://github.com/jamescummings/Bodleian-msDesc-ODD/blob/master/convertTolkien2Bodley.xsl">https://github.com/jamescummings/Bodleian-msDesc-ODD/blob/master/convertTolkien2Bodley.xsl</ref>
+      </fileDesc>
+      <revisionDesc>
+         <change when="2017-07-01">First online publication.</change>
+         <change when="2017-05-29"><persName>James Cummings</persName> Up-converted the markup using <ref target="https://github.com/jamescummings/Bodleian-msDesc-ODD/blob/master/convertTolkien2Bodley.xsl">https://github.com/jamescummings/Bodleian-msDesc-ODD/blob/master/convertTolkien2Bodley.xsl</ref>
          </change>
-         <change when="2017-05-25">
-            <persName>James Cummings</persName> Up-converted the markup using <ref target="https://github.com/jamescummings/Bodleian-msDesc-ODD/blob/master/convertTolkien2Bodley.xsl">https://github.com/jamescummings/Bodleian-msDesc-ODD/blob/master/convertTolkien2Bodley.xsl</ref>
+         <change when="2017-05-25"><persName>James Cummings</persName> Up-converted the markup using <ref target="https://github.com/jamescummings/Bodleian-msDesc-ODD/blob/master/convertTolkien2Bodley.xsl">https://github.com/jamescummings/Bodleian-msDesc-ODD/blob/master/convertTolkien2Bodley.xsl</ref>
          </change>
+         <change when="2020-09-24"><persName>Andrew Dunning</persName> New description.</change>
```

Comment:
@andrew-morrison will it matter if the latest `change` is at the bottom rather than the top? Or is the processing done according to date?

Reply:
If there are no`@when` attributes then the stylesheet will use the first `change` element, as per [the recommendation in the TEI docs](https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-revisionDesc.html). But if at least one `change` has a `@when` then order doesn't matter and it will pick the one with the most recent date (as long as they use a consistent year-month-day format.)